app/src/test/java/com/kinandcarta/create/proxytoggle/feature/tile/ProxyTileServiceTest.kt

```diff
@@ -0,0 +1,99 @@
+package com.kinandcarta.create.proxytoggle.feature.tile
+
+import android.service.quicksettings.Tile
+import com.kinandcarta.create.proxytoggle.android.DeviceSettingsManager
+import com.kinandcarta.create.proxytoggle.model.Proxy
+import com.kinandcarta.create.proxytoggle.stubs.Stubs.PROXY
+import io.mockk.MockKAnnotations
+import io.mockk.confirmVerified
+import io.mockk.every
+import io.mockk.impl.annotations.RelaxedMockK
+import io.mockk.mockk
+import io.mockk.spyk
+import io.mockk.verify
+import org.junit.Before
+import org.junit.Test
+
+class ProxyTileServiceTest {
+
+    @RelaxedMockK
+    lateinit var mockDeviceSettingsManager: DeviceSettingsManager
+
+    @RelaxedMockK
+    lateinit var mockTile: Tile
+
+    private lateinit var subject: ProxyTileService
+
+    @Before
+    fun setUp() {
+        MockKAnnotations.init(this)
+        subject = spyk(ProxyTileService()).apply {
```

Comment:
Not very elegant using a spy, but I need to mock the internal TileService behaviour so I can return a mock Tile